BeeSafe/beesafe-services
# Merged APIs to validate that the user will not be duplicated in mysql…

Combined the insert data API and the get user API.
This will pull data from the email field in the front end and will check it to see if there is a duplicate user in MySQL.
Test:

1. have flask running
2. start your emulator
3. open MySQL workbench
4. go to the emulator and login
5. check duplicate records in workbench
6.try numbers to make sure it doesn't crash
